apps/backend/src/app/modules/maps/maps.service.ts

```diff
@@ -1064,6 +1066,13 @@ export class MapsService {
           ) as Prisma.MMapUpdateInput
         });
         statusChanged = statusHandler[1] !== statusHandler[2];
+
+        // Ensure that discord notification will be sent after update
+        if (statusHandler[2] === MapStatus.PUBLIC_TESTING) {
+          await this.discordWebhookService.sendMapPublicTestingEmbed(tx, map);
+        } else if (statusHandler[2] === MapStatus.APPROVED) {
+          await this.discordWebhookService.sendMapApprovedEmbed(tx, map);
```

Comment:
No, it can't. The only reason I'm using await here is testing, because otherwise we'll have no way of knowing when the webhook service code is done after map status was changed